src/rest_vol_attr.c

```diff
@@ -1614,7 +1614,7 @@ RV_attr_get(void *obj, H5VL_attr_get_args_t *args, hid_t dxpl_id, void **req)
                     char       parent_obj_URI[URI_MAX_LENGTH];
 
                     /* Retrieve the type and URI of the object that the attribute is attached to */
-                    search_ret = RV_find_object_by_path(loc_obj, loc_params->loc_data.loc_by_name.name,
```

Comment:
The VOL passes the name of the object to start from in `loc_params->loc_data.loc_by_name`, so the old way was correct.